lessons/Data-Science/histograms/langs/en-us/pages/notes-histograms.adoc

```diff
@@ -16,8 +16,8 @@ To best understand histograms, it's helpful to contrast them first with bar char
 
 - There’s no strict numerical way to order these bars. 
 
-  * The count of red, yellow and blue balloons would make sense no matter what order they get presented in. 
-  * But *sometimes there’s an order that makes sense*. For example, it would be logical to show the count of t-shirt sizes in order of smallest to largest shirt.
+  * The frequency of red, yellow and blue balloons in the sample would make sense no matter what order they get presented in.
+  * But *sometimes there’s one order that makes sense than the others*. For example, it would be logical to show the number of t-shirt sizes in order of smallest to largest shirt.
```

Comment:
add "more"
But *sometimes there’s one order that makes MORE sense than the others*. For example, it would be logical to show the number of t-shirt sizes in order of smallest to largest shirt.

Reply:
done